integration-tests/src/set.rs

```diff
@@ -36,12 +36,21 @@ async fn test_set_method() -> Result<()> {
         }
     });
 
-    user.call(contract.id(), "set")
+    let set_result = user
+        .call(contract.id(), "set")
         .args_json(args)
         .deposit(parse_near!("0.1 N"))
         .transact()
         .await?
-        .into_result()?;
+        .json::<Value>()?;
+
+    let block_height_json_u64: near_sdk::json_types::U64 = from_value(
+        set_result.get("block_height").unwrap().clone()
+    ).unwrap();
+
+    let block_height: u64 = block_height_json_u64.into();
+
+    assert!(block_height > 0);
```

Comment:
nit: I think it is a bit shorter and easier to read and maintain.

```suggestion
    assert!(
        set_result["block_height"]
            .as_str()
            .unwrap()
            .parse::<u64>()?
            > 0
    );
```